unzip_content.py

```diff
@@ -287,11 +297,19 @@ def get_icon_path(linkJSON):
                     url = link_data['linkUrl']
                     desktop_objects[id].append_localized_url(lang, url)
 
-    apps_path = os.path.join(UNZIP_DIR, 'apps', 'content.json')
+    apps_path = os.path.join(CONTENT_DIR, 'apps', 'content.json')
     apps_file = open(apps_path)
     apps_json = json.load(apps_file)
     apps_file.close()
     for app_data in apps_json:
+        # For the desktop files, we want to prefix and custom Endless
+        # categories with "X-Endless-"
+        categories = app_data['category'].split(';')
+        for i, category in enumerate(categories):
+            if category in X_ENDLESS_CATEGORIES:
+                categories[i] = 'X-Endless-' + category
```

Comment:
OK I see. Why does the appstream XML not support the `X-Endless-` prefix? It seems confusing to me that we have categories specified differently in two places, and would rather address that. If it's too complicated though, or even wrong and I'm missing something, let me know and we can proceed with this as-is.